scripts/generate_textbook.py

```diff
@@ -263,15 +281,22 @@ def _between_symbols(string, c1, c2):
         if link.endswith('.ipynb'):
             yaml_fm += ['interact_link: {}'.format(link.lstrip('./'))]
         yaml_fm += ["title: '{}'".format(title)]
-        yaml_fm += ["permalink: '{}'".format(_prepare_link(link))]
+        page_link = _prepare_link(link)
+        yaml_fm += ["permalink: '{}'".format(page_link)]
         yaml_fm += ['previouschapter:']
         yaml_fm += ['  url: {}'.format(_prepare_link(prev_page_link).replace('"', "'"))]
         yaml_fm += ["  title: '{}'".format(prev_file_title)]
         yaml_fm += ['nextchapter:']
         yaml_fm += ['  url: {}'.format(_prepare_link(next_page_link).replace('"', "'"))]
         yaml_fm += ["  title: '{}'".format(next_file_title)]
+        # In case pre-existing links are sanitized
+        sanitized = page_link.lower().replace('_', '-')
+        if not cased_fs and page_link.lower() == sanitized:
+            raise RuntimeError('Redirect {} clashes with page {} '
+                               'on case-insensitive FS'.format(
+                                   sanitized, page_link))
```

Comment:
What should people do in this situation? Make sure their file names are already-sanitized?

Reply:
Cool - think it makes sense to add a quick sentence along these lines to the error? Or we can add a short description of this problem in the docs and link to it?

Reply:
I've been thinking of how to add a guide for the "build the site locally and push the whole site to the internet" pathway for people...I feel like this should be a part of that info